src/Sarif/Sarif.csproj

```diff
@@ -20,7 +20,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net472;net461;net452</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net452;net462;net472;net48</TargetFrameworks>
```

Comment:
>net462 [](http://example.com/codeflow?start=83&length=6)

So, I've updated our net461 target to net462. Will this cause user issues? Is it helpful to have both to ensure maximal back-compat?

Reply:
Right now, only versions greater than 462 are supported by microsoft ([https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework)).

Based on that, should we keep net452 and net461?

You would only need to add net4xx if you have any special condition for those, otherwise, net framework is able to load those normally.

Reply:
If I've understood you correctly, you've misconstrued the value we're trying to provide, which is to not require developers to recompile their code/update their package references. Make sense? Someone who compiles against net461 can successfully run on a net462 runtime deployment without recompilation. If we prevent those users from consuming our package because our metadata requires they explicitly retarget/recompile, we've broken builds and compromised the .NET compatibility feature. 

And so, it sounds odd, but we should be providing the lowest possible version that allow people to consume it and updates from that version. I'm actually wondering if this might do it.

<TargetFrameworks>netstandard2.0;net45;</TargetFrameworks>

[https://docs.microsoft.com/en-us/dotnet/standard/frameworks](https://docs.microsoft.com/en-us/dotnet/standard/frameworks)

Reply:
OK, so looking at NuGet version data, it appears there are the explicitly specified versions and the inferred compatible versions. If you specify net472, for example, there is an inferred compatibility both with net48 but also down to net461 (there's some hard break at net461 that related to some specific platform, Mono or Silverlight or something).

If I'm correct, specifying net48 would work, except for users who tried to drop down to net45. I'm still going to proceed with and test net45, which I believe will imply compatibility from that version all the way through net48.